### ISSUE TITLE
Structure function scientific tutorial

### ISSUE THREAD

**Issue by nevencaplar** [lincc-frameworks/tape#159]
Write tutorial geared towards scientists on how to use structure function:

1. Quick Start
2. How to use different SF calculators
3. Young stellar object example
Stretch goal:
4. Custom SF
5. AGN Stripe 82

**Comment by nevencaplar** [comment]
Blocked by #180 and #181

**Comment by nevencaplar** [comment]
Blocked by #198
